posts/indu-plp-part1/plp-part1.qmd

```diff
@@ -0,0 +1,217 @@
+
+4. Feature Construction
+Highlights **constructing features** from observational data (e.g., clinical conditions, demographics) for model input.
+
+5. Model Training and Evaluation
+Involves **training models** with machine learning algorithms and **evaluating** their performance using metrics like AUC and cross-validation.
+
+6. Statistical Methods for Model Evaluation
+Uses **statistical metrics** such as AUC, calibration, and decision curve analysis to assess model performance and clinical utility.
+
+7. Reproducibility and Transparency
+Ensures **reproducibility and transparency**, documenting all steps to facilitate consistent and reproducible model development across settings.
+
+# Research Question
+
+The core research question we aim to address in this PLP pipeline is:
+
+> **Among patients diagnosed with hypertension, who will go on to develop diabetes?**
+
+This question is of paramount importance in clinical decision-making, as it helps identify high-risk individuals who may benefit from early intervention to prevent or manage diabetes. The challenge in answering this question lies in accurately identifying the patients with hypertension who are at risk of progressing to diabetes, based on their medical history and risk factors.
+
+## Cohort Construction
+
+Cohorts are groups of patients defined by specific criteria that are relevant to the research question. For this task, two main cohorts need to be defined:
+
+- **Target Cohort**: This refers to the group of patients we want to make predictions for. In our case, it includes patients who have been diagnosed with hypertension. These patients serve as the starting point for our prediction timeline.
+
+- **Outcome Cohort**: This refers to the clinical event we aim to predict. In our case, it includes patients from the target cohort who are subsequently diagnosed with diabetes within a specified time window. This event marks the outcome that our model will learn to forecast.
+
+These cohort definitions are central to structuring the data pipeline, as they form the foundation for downstream tasks like feature extraction, model training, and evaluation.
+
+# Defining Cohorts using OHDSICohortExpressions.jl
+
+In the context of this research, I received a 20GB synthetic dataset that contains 1,115,000 fake patients (1,000,000 alive and 115,000 deceased), each with 3 years of medical history. This dataset was provided as a DuckDB database, a lightweight, high-performance analytical database that allows fast querying of large datasets directly from local files without the need for a server.
+
+For cohort creation, I used OHDSI cohort definitions provided directly by my mentor in the form of two JSON files: **Hypertension.json** (for the target cohort) and **Diabetes.json** (for the outcome cohort). To execute them, I used the OHDSICohortExpressions.jl package to convert the JSON definitions into SQL queries, which were then run against the DuckDB database to extract the relevant cohorts.
```

Comment:
Could you link the OHDSICohortExpressions.jl package? Also, could you make the target and outcome cohorts as a markdown list here? I think that would be more clear.